src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,28 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int MIN_COUNT_SYMBOL = 6;
+    public static final int VALID_AGE = 18;
```

Comment:
The constant names `MIN_COUNT_SYMBOL` and `VALID_AGE` should be more descriptive to reflect their purpose. For example, `MIN_PASSWORD_LENGTH` and `MIN_USER_AGE` would be more informative.